.github/workflows/reusable_proxy_lib.yml

```diff
@@ -59,11 +59,9 @@ jobs:
       - name: Build UMF
         run: cmake --build ${{env.BUILD_DIR}} -j $(nproc)
 
-      # TODO enable the provider_file_memory_ipc test when the IPC tests with the proxy library are fixed
-      # see the issue: https://github.com/oneapi-src/unified-memory-framework/issues/864
       - name: Run "ctest --output-on-failure" with proxy library
         working-directory: ${{env.BUILD_DIR}}
-        run: LD_PRELOAD=./lib/libumf_proxy.so ctest --output-on-failure -E provider_file_memory_ipc
+        run: LD_PRELOAD=./lib/libumf_proxy.so ctest --output-on-failure
 
```

Comment:
there's another one down below, pointing to the same issue

Reply:
@lukaszstolarczuk Fixed.
Done.